docker-compose.yml

```diff
@@ -31,13 +33,31 @@ x-common: &common
         memory: 3G
 
 services:
-  ingress: # Nginx ingress for all hostnames. Change nginx_ingress.conf to add more hostnames / routes
+  ingress:
+    # Nginx ingress for all hostnames. Change nginx_ingress.conf to add more hostnames / routes
     extends:
       file: docker-compose.dev.yml
       service: ingress
     <<: *common
 
-  autoheal: # Restarts containers when they become unhealthy
+  db:
+    image: postgres
+    restart: unless-stopped
+    ports:
+      - 5432:5432
+    environment:
+      POSTGRES_PASSWORD: postgres
+    volumes:
+      - ./comms/discovery/db/docker-initdb:/docker-entrypoint-initdb.d
+      - ./dev-tools/startup/init-db.sql:/docker-entrypoint-initdb.d/init-db.sql
+    # <<: *common
```

Comment:
curious why this is commented out. I like having it on every service because it standardizes the max memory so when you do `docker stats` you can see all the percentages and compare apples to apples (otherwise one container says like 1GB/12GB and that's different percentage from another container that's using 1GB/3GB)

Reply:
Oops, this was for testing, will uncomment, thanks